moveit/package.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0"?>
+<package>
+  <name>moveit</name>
+  <version>0.0.0</version>
```

Comment:
we should set this before merging (based on your issue, thanks for opening that)